config/2018.ts

```diff
@@ -104,13 +104,20 @@ const From2018: From2018 = {
       url: 'https://mechanicalrock.io/',
     },
     {
-      id: 'yow-perth',
+      id: 'yow-perth-keynote',
       imageUrl: '/static/images/sponsors/yow-perth.png',
       name: 'YOW! Perth',
       serviceProvided: 'Keynote',
       type: SponsorType.Service,
       url: 'http://west.yowconference.com.au/',
     },
+    {
+      id: 'bankwest',
```

Comment:
Why did this need to be added?

Reply:
According to the 2018 Agenda, Bankwest sponsored the lunchnote. Didn't know what category to add them. This category meant I could use them in the agenda without them appearing in the list of sponsors.